src/ConversationList/ConversationListBox.vala

```diff
@@ -102,6 +102,32 @@ public class Mail.ConversationListBox : VirtualizingListBox {
                 conversation_selected (((ConversationItemModel) row).node);
             }
         });
+
+        button_press_event.connect ((e) => {
```

Comment:
AFAIK are we using `button_release_event` for such things in other places. This has the benefit of not interfering with drag and drop for example. So I think its a good idea to stick to the `some_key_release` events if there's no explicit reasoning for `some_key_press`.

Reply:
This is fixed